db/migrate/20200511091734_create_aws_texts.rb

```diff
@@ -0,0 +1,9 @@
+class CreateAwsTexts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :aws_texts do |t|
+      t.string :title
+      t.string  :content
```

Comment:
`content`カラムは文字の量が多くなりますので，`text`型にしておきましょう。

```suggestion
      t.text  :content
```

まだ，データがありませんので，この修正を行った上でターミナルから次を実行して下さい。これでデータベースも修正されます。

```
rails db:migrate:reset
```

Reply:
型の修正とマイグレーションのリセットがおわりましたので、
再度レビューをお願いします。

Reply:
すいません、不要なブランクが含まれていました。
再度レビューをおねがいします！